src/bandersnatch/tests/test_mirror.py

```diff
@@ -492,7 +492,7 @@ def test_validate_todo(mirror: Mirror) -> None:
 @pytest.mark.asyncio  # type: ignore
 async def test_cleanup_non_pep_503_paths(mirror: Mirror) -> None:
     raw_package_name = "CatDogPython69"
-    package = Package(raw_package_name, 11, mirror)
+    package = Package(raw_package_name, mirror)
```

Comment:
Is this just to test serial = 0 default now?

Reply:
Flipped that unintentionally, I fixed it